compliance/cis_k8s/rules/cis_4_2_12/test.rego

```diff
@@ -0,0 +1,34 @@
+package compliance.cis_k8s.rules.cis_4_2_12
+
+import data.kubernetes_common.test_data
+import data.lib.test
+
+test_violation {
+	test.assert_fail(finding) with input as rule_input("")
+	test.assert_fail(finding) with input as rule_input("--feature-gates=RotateKubeletServerCertificate=false")
+	test.assert_fail(finding) with input as rule_input_with_external("", create_process_config(false, false))
+	test.assert_fail(finding) with input as rule_input_with_external("--feature-gates=RotateKubeletServerCertificate=false", create_process_config(false, false))
+	test.assert_fail(finding) with input as rule_input_with_external("--feature-gates=RotateKubeletServerCertificate=false", create_process_config(true, false))
+}
```

Comment:
what about:
```suggestion
	test.assert_fail(finding) with input as rule_input_with_external("--feature-gates=RotateKubeletServerCertificate=false", create_process_config(true, true))
}
```
should it fail in this case? is this case even valid?

Reply:
legit ! and was added!